packages/entities/entities-shared/src/components/entity-base-config-card/ConfigCardItem.vue

```diff
@@ -103,14 +103,26 @@
               />
             </div>
 
-            <span
-              v-else
-              class="attrs-data-text"
-              :class="{ 'truncated': truncated }"
-              :title="truncated ? item.value : undefined"
-            >
-              {{ componentAttrsData.text }}
-            </span>
+            <div v-else>
+              <KTooltip
```

Comment:
TBH, I think the better solution here would be to utilize a dynamic component or something that would/would not render the tooltip instead of injecting elements into the DOM.

Alternatively, we could also alter KTooltip to not render something if the `label` is `undefined`

Reply:
I updated it to not render things in the DOM, I figured out the break point where the label starts being truncated